gen/_template/schema/struct.tmpl

```diff
@@ -6,7 +6,12 @@
 	type {{ $.Name }} struct {
 	{{- range $f := $.Fields }}
 		{{- template "godoc" $f.GoDoc }}
-		{{ $f.Name }} {{ $f.Type.Go }} {{ if $f.Tag.JSON }}{{ printf "json:%q" $f.Tag.JSON | backquote }}{{ end }}
+		{{ $f.Name }} {{ $f.Type.Go -}}
+		{{- if $f.Tag.JSON }}
+			{{- append $f.FieldTags (printf "json:%q" $f.Tag.JSON) | join " " | backquote }}
+		{{- else if len $f.FieldTags }}
```

Comment:
It is better to move this logic to the special method, like `GoTags`.

Reply:
In this case, the `gen/ir/field.go:Field.FieldTags` might be moved to `gen/ir/field.go:Tag.CustomTag`, for example:
```go
// Tag of Field.
type Tag struct {
	JSON       string             // json tag, empty for none
	Form       *openapi.Parameter // query form parameter
	CustomTags []string           // CustomTags is a list of additional struct field tags
}
```
The receiver of `GoTags` would be `*Tag`
Of course, `x-ogen-field-tags` would be renamed to `x-ogen-custom-field-tags` and `jsonschema/schema.go:Schema.FieldTags` to `CustomFieldTags`.

What do you think?

Reply:
> The receiver of `GoTags` would be `*Tag`

Sounds good to me.

> Of course, `x-ogen-field-tags` would be renamed to `x-ogen-custom-field-tags` and `jsonschema/schema.go:Schema.FieldTags` to `CustomFieldTags`.

Probably, we should use the same name as deepmap. It would make migration from one generator to another easier.